test/Header/SetCookieTest.php

```diff
@@ -343,6 +343,12 @@ public function testDoesNotAcceptCookieNameFromArbitraryLocationInHeaderValue()
         $this->assertNotEquals('leo_auth_token', $setCookieHeader->getName());
     }
 
+    public function testEmptyExpiresSection()
+    {
+        $cookie = 'Set-Cookie:path=/; Expires=; HttpOnly;';
+        $setCookieHeader = SetCookie::fromString($cookie);
```

Comment:
> Please assert `getExpires` returns the correct value.

I think that is a test on its own. One test to see there is no exception thrown and another test to see if `getExpires` returns `null`.